htmlhint-server/src/server.ts

```diff
@@ -60,6 +65,74 @@ function getRange(error: htmlhint.Error, lines: string[]): any {
     };
 }
 
+function loadRules(HTMLHint: any, force: boolean): any {
```

Comment:
for consitency, I'd call `HTMLHint` `linter` or `htmlHintLinter` - something to indicate that it is an instance ofa linter.

Reply:
Or can you just use the module var `linter` here?